fitting/fitScanData.py

```diff
@@ -337,7 +337,7 @@ def fit(self, debug=False):
                                     )
 
                     # Fit
-                    fitResult = self.scanHistos[vfat][ch].Fit('myERF','SQ')
+                    fitResult = self.scanHistos[vfat][ch].Fit('myERF','SQ') 
```

Comment:
trailing white space here, this will generate spurious `git diff`'s that will be hard to track down.  

I would suggest configuring your editor to how to strip trailing whitespaces on saving the file [example in vim](http://vim.wikia.com/wiki/Remove_unwanted_spaces).  Similar examples exist for `emacs`.

Reply:
thx